### PR TITLE
LPAL-722 Roll-up dependabot updates and fix issues

### PR DESCRIPTION
**Pull request by townxelliot**
## Purpose

[LPAL-722](https://opgtransform.atlassian.net/browse/LPAL-722)

Fixes LPAL-722
Incorporates LPAL-734 For these containers.

## Approach

Incorporates dependabot PRs:

* https://github.com/ministryofjustice/opg-lpa/pull/870
* https://github.com/ministryofjustice/opg-lpa/pull/869
* https://github.com/ministryofjustice/opg-lpa/pull/868

Also includes a few code fixes to make the above work correctly, a rollback of localstack which was causing instabilities, and updates to composer and npm config.

Suggest we close this one - https://github.com/ministryofjustice/opg-lpa/pull/840 - as PHP 8.1 is not compatible with Laminas.

I have also not incorporated https://github.com/ministryofjustice/opg-lpa/pull/859, which updates cypress. This causes the CI build to fail, as outlined in the ticket (LPAL-722).

I've run unit tests and cypress tests locally and they are passing OK.

## Learning

n/a

## Checklist

* [ ] I have performed a self-review of my own code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] I have added mandatory tags to terraformed resources, where possible
* [ ] If I added a package.json or composer.json, I also made sure this is included in the script in `.github/workflows/dependabot-update.yml`
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-front/module/Application/tests/Model/Service/Session/FilteringSaveHandlerTest.php [hunk 1]
```diff
@@ -195,6 +195,6 @@ public function testDestroy()
     public function testGc()
     {
         $redisMock = $this->makeSaveHandlerWithMock('tcp://barhost:6737');
-        $this->assertSame(TRUE, $this->saveHandler->gc(1));
+        $this->assertSame(1, $this->saveHandler->gc(1));
     }
 }
```
Comment (Raffers) [inline service-front/module/Application/tests/Model/Service/Session/FilteringSaveHandlerTest.php:200]: You no longer have the pre-commit hooks installed?

**Review by Raffers — APPROVED** [review]

**Review by williamfalconeruk — CHANGES_REQUESTED** [review]
As per comment re critical bug.

**Comment by williamfalconeruk** [comment]
LPAL-731 raised to address this vuln first.

**Review by williamfalconeruk — APPROVED** [review]

**Comment by williamfalconeruk** [comment]
This has been fixed in front, so we can proceed.

**Review by williamfalconeruk — APPROVED** [review]

**Review by williamfalconeruk — APPROVED** [review]

**Pull request merged**